include/readout/models/ReadoutModel.hpp

```diff
@@ -252,9 +253,10 @@ class ReadoutModel : public ReadoutConcept
 
           if (m_fake_trigger) {
             dfmessages::DataRequest dr;
-            dr.trigger_timestamp = timesyncmsg.daq_time > 500 * us ? timesyncmsg.daq_time - 500 * us : 0;
-            auto width = 10000;
-            uint offset = 100;
+           
+            dr.trigger_timestamp = timesyncmsg.daq_time > 1 ? timesyncmsg.daq_time + 1 : 0;
```

Comment:
This completely breaks the fake trigger functionality for WIBs and DAPHNE. PACMAN should move to nanoseconds, and this fake trigger should be configurable. But don't overwrite defaults, as it breaks stuff runtime for emulators.